hangman/lib/hangman/game.ex

```diff
@@ -0,0 +1,75 @@
+defmodule Hangman.Game do
+    defstruct(
+        turns: 7,
+        game_state: :initializing,
+        used: %MapSet{},
+        word: [],
+        last_guess: "",
+    )
+
+  def new_game() do
+    word = String.codepoints( Dictionary.random_word() )
```

Comment:
## idiomatic -2

I'd really like to see a pipeline here:

~~~ elixir
word = Dictionary.random_word()
       |> String.codepoints()
       |> Enum.drop(-1)
~~~

## -5 works

The `Enum.drop` is a bug. If you run this against my tests or against the human player code, it drops the last letter of the word to be guessed. I'm guessing you originally tested this by doing an IO.gets on the word to be guessed, and this code stripped the trailing \n?